examples/tutorial_for_R/R_model_tutorial.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
Can you turn this tutorial into an integration test? Check out the [PySpark integration test](https://github.com/ucbrise/clipper/blob/develop/integration-tests/deploy_pyspark_models.py) for an example. Also, add a README.md in the `containers/R` directory that discusses how to deploy an R model, any restrictions on the model inputs or outputs, and the dependencies needed (both Python dependencies like `rpy2` and R dependencies.